README.md

```diff
@@ -162,15 +175,15 @@ In the sandbox configure a cashbox based on the list of supported packages below
 | Name                                           | Versions    |
 | ---------------------------------------------- | ----------- |
 | fiskaltrust.Middleware.Queue.MySQL             | v1.3.37     |
-| fiskaltrust.Middleware.Queue.SQLite            | v1.3.38-ci-22221-55477     |
+| fiskaltrust.Middleware.Queue.SQLite            | v1.3.38-rc1-22249-56008     |
 | fiskaltrust.Middleware.SCU.DE.FiskalyCertified | v1.3.35     |
 | fiskaltrust.Middleware.SCU.DE.CryptoVision     | v1.3.34 |
 | fiskaltrust.Middleware.SCU.DE.DeutscheFiskal   | v1.3.35     |
 | fiskaltrust.Middleware.SCU.DE.DieboldNixdorf   | v1.3.20     |
 | fiskaltrust.Middleware.SCU.DE.Epson            | v1.3.19     |
 | fiskaltrust.Middleware.SCU.DE.Swissbit         | v1.3.35-rc1    |
 | fiskaltrust.Middleware.SCU.DE.SwissbitCloud    | v1.3.35     |
-| fiskaltrust.Middleware.Helper.Helipad          | v1.3.26     |
+| fiskaltrust.Middleware.Helper.Helipad          | v1.3.33     |
```

Comment:
I think we can remove this table as it is also present above

Reply:
will do